package.json

```diff
@@ -146,7 +146,7 @@
     "vue-class-component": "^6.1.1",
     "vue-loader": "^15.2.4",
     "vue-template-compiler": "^2.5.16",
-    "webpack": "^5.0.0-alpha.5"
```

Comment:
I'm hesitant about moving to webpack 5 until it's properly out of beta; it's been in alpha / beta for a year now and so it may not be soon

Reply:
I did move **from** webpack 5.0.0 **to** webpack 4.0.0 :) I think it's ok, right?

Reply:
I may have misread that 😉

Yeah webpack 4 is fine for now. Webpack 5 still has a ways to go I think